app/serializers/user_serializer.rb

```
@@ -133,6 +134,26 @@ def website
+    return if website_host.nil?
+    if website_host == discourse_host
+      # example.com == example.com
+      website_host + URI(website.to_s).path
+    elsif (website_host.split('.').length == discourse_host.split('.').length) && discourse_host.split('.').length > 2
+      # www.example.com == forum.example.com
+      website_host.split('.')[1..-1].join('.') == discourse_host.split('.')[1..-1].join('.') ? website_host + URI(website.to_s).path : website_host
+    else
+      # example.com == forum.example.com
+      discourse_host.ends_with?("." << website_host) ? website_host + URI(website.to_s).path : website_host
+    end
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `website_name` is too high. [<2, 35, 12> 37.05/23]

(Metrics/AbcSize)